### DISCUSSION TITLE
《阿凡達2》後有一項關鍵技術來自台灣七年級生

### DISCUSSION THREAD

**Discussion by benestia** [1101-GS4719-C/Python-Community-GS4719-C#1060] [科技新知]
阿凡達2終於上映，記得10年前看第一集的時候有種被科技的威力震懾的感覺，時隔十年再看第二集還是有同樣的感覺。知道幕後的特效有台灣世新大學智慧攝影棚的助力很為台灣的電影產業開心！以下是相關內容的連結~有看過的人可以藝起來分享討論觀影心得嗎~

https://www.gvm.com.tw/article/86742
https://finance.ettoday.net/news/2163962

還有這裡是阿凡達2的介紹~不知道阿凡達的人也可以從這裡了解~

https://www.gq.com.tw/entertainment/article/avatar-the-way-of-water

**Comment by ru0372572** [comment]
還沒看第二集，考完期末一定要去看!!真心覺得科技進步的速度比想像中的快太多，已經不是空有想法卻無法實踐的年代了
只記得小時候不敢看電視播的阿凡達，家人在看的時候都要躲到房間，而且時不時會覺得家裡的轉角過去就會遇到一隻，超怕XD

**Comment by Valentinaa0214** [comment]
沒看過第一集也沒看過第二集，
小時候覺得他們一隻藍藍的長的超噁心哈哈哈哈。
光是透過預告片就可以讓我身歷其境，進入一個充滿奇幻、想像力的世界，
創造驚人的視覺效果，這就是一個十分成功的電影特效吧。

**Comment by Tuoresa** [comment]
放假要去看第二集~
好久以前看第一集的時候就非常喜歡！
裡面的特效與壯觀的畫面對於當時的我來說真的太震撼了，
故事的闡述也很滿足我對天馬行空的熱愛哈哈
一直期待著還有第二集，終於上映了

**Comment by dinosaur1009** [comment]
台灣的電影技術產業發展越來越厲害是一件值得期待的事！還能在國際上展露頭角

**Comment by ABCDEFGHIJKLMNO12345678** [comment]
雖然第 二集都沒看過 但僅僅看了預吿片就被他驚人的視覺特效甚深吸引 可說是相當不錯的一部電影

**Comment by Twenty0522** [comment]
沒有看過第一集跟第二集，但影評很好，友人推薦一定要看3D版呈現出的極美動畫。這篇文的連結除了有電影的吸睛點之外，還進一步探究背後的技術，以及電影與台灣的相關性，幫助我們深入了解除了電影以外的事情。

**Comment by BlahFish1006** [comment]
看過無數次第一集，想趕快考完期末考去看第二集 QQ
印象中之前看過一篇報導，說第二集使用到的技術過於先進高級，各國有些電影院還沒辦法有能力播出最原版，
只能退一步求其次，還有日本電影院出來道歉，真的太讓人期待了啊~!
除了拍攝電影的技術，我一直對於電影中主角使用的潛行技術深感興趣，他本身是一位下肢癱瘓的軍人，
但潛入進娜美人的身體後卻可以自由行動，若有朝一日這這項技術能在現實中被實現，那感覺會成為醫療中的一項創舉。

**Comment by Sylvie0601** [comment]
沒看過第一集也沒看過第二集的人+1。
但我覺得阿凡達第一集的時候就給大家帶來非常大的震撼，導致影迷都念念不忘，在當時，身旁的朋友也對於阿凡達這部電影十分喜愛，紛紛都認為這部電影帶給他們的張力含震撼是無法用言語來形容的

**Comment by harrylee1971** [comment]
阿帆達真的很酷，他的背後技術肯定是CGI特效的頂尖水準，這輩子有這電影看真的很棒
